forge-gui/res/cardsfolder/upcoming/phantasmal_extraction.txt

```diff
@@ -0,0 +1,11 @@
+Name:Phantasmal Extraction
+ManaCost:1 B
+Types:Sorcery
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast if you weren't the starting player.
+SVar:X:Count$StartingPlayer.0.1
+A:SP$ Reveal | ValidTgts$ Opponent | RevealAllValid$ Card.nonLand+TargetedPlayerOwn+cmcLE4 | SubAbility$ DBPumpAll | RememberRevealed$ True | SpellDescription$ Target opponent reveals each nonland card in their hand with mana value 4 or less. Choose a card revealed this way or a card in their graveyard. Exile that card.
```

Comment:
Probably break this SpellDescription$ up

Make sure StackDescription isn't a mess